.github/workflows/examples_CI.yml

```diff
@@ -0,0 +1,35 @@
+name: Examples CI
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        version:
+          - '1' # test always on the latest released version 
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+    steps:
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v2
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+      - uses: julia-actions/cache@v2
+      - name: Install dependencies
+        run: julia --project=examples/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+      - name: Run Soil Model Heat Column 
+        run: julia --project=examples/ examples/soil_heat_column.jl
+      - name: Run Soil Model Heat Global 
+        run: julia --project=examples/ examples/soil_heat_global.jl
+      - name: Run Soil Model Heat Global ERA5 
+        run: julia --project=examples/ examples/soil_heat_global_era5.jl
```

Comment:
Can we temporarily omit this one? We'll need to add an automatic data download to make it work, and this will come in a later PR.